julia.spdx.json

```diff
@@ -446,11 +434,6 @@
             "relationshipType": "BUILD_DEPENDENCY_OF",
             "relatedSpdxElement": "SPDXRef-JuliaMain"
         },
-        {
-            "spdxElementId": "SPDXRef-JuliaStatistics",
-            "relationshipType": "BUILD_DEPENDENCY_OF",
-            "relatedSpdxElement": "SPDXRef-JuliaMain"
-        },
```

Comment:
I think technically we still want to include Statistics in the spdx since we ship it?

Reply:
Maybe. I copied what was done for DelimitedFiles at https://github.com/JuliaLang/julia/pull/45540. But when that PR was merged DelimitedFiles was supposed to be removed so maybe it should be added back.